addons/skin.estuary/language/resource.language.de_de/strings.po

```diff
@@ -186,7 +187,7 @@ msgstr "Wiedergabelisten-Einstellungen"
 
 msgctxt "#31043"
 msgid "Set the type and add rules to create a smart playlist. These playlists are dynamic and include all media items from your database which apply to your chosen rules."
-msgstr "Typ einstellen und Regeln für die intelligente Wiedergabeliste hinzufügen. Diese Art von Wiedergabeliste ist dynamisch und beinhaltet alle Medien aus der Bibliothek, die die gesetzten Regeln erfüllen."
+msgstr "Typ einstellen und Regeln für die intelligente Wiedergabeliste hinzufügen. Diese Art von Wiedergabeliste ist dynamisch und beinhaltet alle Medien aus der Bibliothek, die gesetzten Regeln erfüllen."
```

Comment:
@gade this change is wrong. In German it grammatically sometimes correct to have the same word twice in a row. But better style would be: "Bibliothek, welche die"

Reply:
Thanks.
I'll change it.